src/PantherTestCaseTrait.php

```diff
@@ -147,7 +147,7 @@ public static function isWebServerStarted(): bool
      *
      * @param array $options see {@see $defaultOptions}
      */
-    protected static function createPantherClient(array $options = [], array $kernelOptions = [], array $managerOptions = []): PantherClient
+    protected static function createPantherClient(array $options = [], array $kernelOptions = [], array $managerOptions = [], array $browserArguments = null): PantherClient
```

Comment:
```suggestion
    protected static function createPantherClient(array $options = [], array $kernelOptions = [], array $managerOptions = [], array $browserArguments = []): PantherClient
```
??

Reply:
yes, passing an empty array will not break anything

Reply:
Done